src/sentry/web/client_config.py

```diff
@@ -96,7 +96,7 @@ def _get_public_dsn() -> str | None:
     if settings.IS_DEV and not settings.SENTRY_USE_RELAY:
         return ""
 
-    project_id = settings.SENTRY_FRONTEND_PROJECT or settings.SENTRY_PROJECT
+    project_id: Any | None = settings.SENTRY_FRONTEND_PROJECT or settings.SENTRY_PROJECT
```

Comment:
this should be `int | None` -- and in theory should infer it from the typings already in settings ? (I don't think this annotation is necessary)

Reply:
Without this annotation `if project_id is None:` line throws `unreachable`